releases/release-8.0.0.md

```diff
@@ -0,0 +1,571 @@
+---
+title: TiDB 8.0.0 Release Notes
+summary: Learn about the new features, compatibility changes, improvements, and bug fixes in TiDB 8.0.0.
+---
+
+# TiDB 8.0.0 Release Notes
+
+Release date: xx xx, 2024
+
+TiDB version: 8.0.0
+
+Quick access: [Quick start](https://docs.pingcap.com/tidb/v8.0/quick-start-with-tidb) | [Installation packages](https://www.pingcap.com/download/?version=v8.0.0#version-list)
+
+8.0.0 introduces the following key features and improvements:
+
+<table>
+<thead>
+  <tr>
+    <th>Category</th>
+    <th>Feature/Enhancement</th>
+    <th>Description</th>
+  </tr>
+</thead>
+<tbody>
+  <tr>
+    <td rowspan="4">Scalability and Performance</td>
+    <td><a href="https://docs.pingcap.com/tidb/v8.0/pd-microservices">Disaggregation of PD to improve scalability (experimental)</a> </td>
+    <td>Placement Driver (PD) contains multiple critical modules to ensure the normal operation of TiDB clusters. As the workload of a cluster increases, the resource consumption of each module in PD also increases, causing mutual interference between these modules and ultimately affecting the overall service quality of the cluster. To address this issue, starting from v8.0.0, TiDB supports disaggregating the TSO and scheduling modules of PD into independently deployable microservices, which can significantly reduce the mutual interference between modules as the cluster scales. With this architecture, much larger clusters with much larger workloads are now possible for TiDB.</td>
```

Comment:
```suggestion
    <td>Placement Driver (PD) contains multiple critical modules to ensure the normal operation of TiDB clusters. As the workload of a cluster increases, the resource consumption of each module in PD also increases, causing mutual interference between these modules and ultimately affecting the overall service quality of the cluster. Starting from v8.0.0, TiDB addresses this issue with disaggregation of the TSO and scheduling modules in PD into independently deployable microservices. This can significantly reduce the mutual interference between modules as the cluster scales. With this architecture, much larger clusters with much larger workloads are now possible.</td>
```

Reply:
@qiancai PTAL

Reply:
LGTM